apps/mobile/src/components/widgets/components/widget/widget-title.tsx

```diff
@@ -0,0 +1,25 @@
+import { useTheme } from '@shopify/restyle';
+
+import { Box, ChevronRightIcon, Text, Theme } from '@leather.io/ui/native';
+
+import { FiatBalance } from '../balance/fiat-balance';
+
+interface WidgetTitleProps {
+  title: string;
+  totalBalance?: string;
```

Comment:
Shouldn't we be passing `Money`?

Reply:
We should, good spot. Right now it's a mocked string though. 

I'll add a note to update in my next work